### ISSUE TITLE
"Close" button is not accessible via keyboard in cursor mode of 'Query1' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1038]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1240929

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure..
Navigate to "New SQL Query" menu item and select it.
Navigate through all the controls under Query 1.
Turn on JAWS and navigate to query 1 tab and try to access "Close(X)" button of Query 1 tab.
Verify the issue.

Actual Result:
"Close" button is not accessible via keyboard in scan mode of 'Query1' blade.

Expected Result:
"Close" button should be accessible via keyboard in scan mode of 'Query1' blade.

Note:
This issue does not exist with Narrator and NVDA.
Similar issue also exists for similar controls under 'Data explorer' screen for created "Azure Cosmos DB for MongoDB API".

https://user-images.githubusercontent.com/8519407/132718834-40ba75bc-fdc4-4f7c-b4a8-448aeb5919a1.mp4
